.github/workflows/ci.yml

```diff
@@ -2,7 +2,7 @@ name: Tests
 
 on:
   push:
-    branches: [ "main" ]
+    tags: [ "v[0-9]+.[0-9]+.[0-9]+" ]
```

Comment:
This is a bit confusing. It looks like we will not be running part of the CI in push to main anymore. My understanding was, we like to keep everything as is but add a step `wheels` on push of tags.

Reply:
Canceled that changes and moved `tags` condition to `wheels.yml`